registration-processor/pre-processor/registration-processor-cmd-validator-stage/README.md

```diff
@@ -0,0 +1,26 @@
+# registration-processor-cmd-validator-stage
+
+This component validates the User, Machine, Centre details from the Packet
+
+## Design
+
+[Design - Approac for User, Machine, Center Validation](https://github.com/mosip/registration/blob/master/design/registration-processor/Approach_for_umc_validation.md)
+
+## Default Context-path and Port
+```
+server.port=8089
```

Comment:
changed these properties for all stages

Reply:
Please remove the spring properties since those will be used only by the mosip executor and not the stages. Remove the below

spring.cloud.config.uri=localhost
spring.cloud.config.label=master
spring.profiles.active=mz

Reply:
removed these 3 properties.